pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -106,365 +26,70 @@ task BuildStarSingleNucleus {
     set -eo pipefail
 
     python3 /script/modify_gtf.py  \
-    --input-gtf ~{references.annotation_gtf} \
+    --input-gtf ~{annotation_gtf} \
     --output-gtf ~{annotation_gtf_modified} \
```

Comment:
Are these always named `"modified_gencode.v~{gtf_version}.primary_assembly.annotation.gtf"`?

Probably we should at minimum remove the "gencode" part

Reply:
Removed gencode and primary.assembly